### PR TITLE
Added description in README file.

### PR DESCRIPTION
**Pull request by patrycja93**
Completed a README file about necessary data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+## Motivation
+
+The idea of this project is to help with understanding and consolidate knowledge about different technologies used in programming. 
+
+## About project
+
+### Aim
+The project was created to learn and practice using these skills in a real platform with real problems. 
+
+The `Animal shelter` is an application that focuses on simple operations in the beginning and in process of time will be developed and new functionalities will be added.
```
Comment (afornalik) [inline README.md:12]: This sentence is too long. Could you please create two of this one? Maybe "... that initially focuses on simple operations. With time new functionalities will be added."

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+## About project
+
+### Aim
+The project was created to learn and practice using these skills in a real platform with real problems. 
+
+The `Animal shelter` is an application that focuses on simple operations in the beginning and in process of time will be developed and new functionalities will be added. 
+
+### The main goal
+The `Animal shelter` stores data about animals and employees (zookeepers, doctors etc). Both animals and employees can be added, delete, update, and read data.
```
Comment (afornalik) [inline README.md:15]: Please add dot ->"etc."

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+### Aim
+The project was created to learn and practice using these skills in a real platform with real problems. 
+
+The `Animal shelter` is an application that focuses on simple operations in the beginning and in process of time will be developed and new functionalities will be added. 
+
+### The main goal
+The `Animal shelter` stores data about animals and employees (zookeepers, doctors etc). Both animals and employees can be added, delete, update, and read data. 
+
+### Technologies
+In this project the following technologies will be used:
```
Comment (afornalik) [inline README.md:18]: Please add coma "In this project,"

File: README.md [hunk 4]
```diff
@@ -1 +1,31 @@
...
+*Java 11, Spring, H2 database (in the beginning), Mockito, Junit.*
```
Comment (afornalik) [inline README.md:19]: Also add Maven, Hibernate(I think that we will use it) here.

File: README.md [hunk 5]
```diff
@@ -1 +1,31 @@
-# animal-shelter
\ No newline at end of file
+# Animal shelter
```
Comment (afornalik) [inline README.md:1]: I am not sure about the aim of these three paragraphs "Motivation, Aim, The main goal" maybe we should consider adding content of these paragraphs under this header - something like what we do and why. What do you think?

File: README.md [hunk 6]
```diff
@@ -1 +1,31 @@
...
+
+The technologies were chosen because of their wide use in the market.
+
+## How to run the project
+*TBD*
+
+## How to run tests
+*TBD*
+
+
```
Comment (afornalik) [inline README.md:29]: Maybe we can add authors paragraph here.

**Review by afornalik — COMMENTED** [review]
Overall looks fine. Thanks for creating the document.

File: README.md (hunk 6 above)
Comment (patrycja93) [reply README.md:31]: Do you think is it necessary?

File: README.md (hunk 1 above)
Comment (patrycja93) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (patrycja93) [reply README.md]: Ok, done.

File: README.md (hunk 3 above)
Comment (patrycja93) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (patrycja93) [reply README.md]: Thanks, done.

File: README.md (hunk 5 above)
Comment (patrycja93) [reply README.md:1]: Corrected.

File: README.md (hunk 6 above)
Comment (afornalik) [reply README.md:31]: It was just a proposition. We can stay without it.

**Review by afornalik — APPROVED** [review]

**Comment by afornalik** [comment]
Hi Patrycja, just a reminder. For me it looks good we can merge it when you are ready.

**Comment by patrycja93** [comment]
Thank you Andrzej, I'll merge.

**Pull request merged**